src/schemas.py

```
@@ -220,3 +220,19 @@ def validate_queries(cls, v):
 
 class DialecticResponse(BaseModel):
     content: str
+
+
+class DeriverStatus(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)